docs/docs/pods.md

```diff
@@ -538,7 +507,7 @@ The following pod definition can serve as a reference to create more complicated
           "executable": false,
           "extract": true,
           "cache": true,
-          "destPath": "newname.zip"
+          "destPath": "/path/to/archive.zip"
```

Comment:
@nfnt what are the rules here for the mesos fetcher? can any, fully-qualified, non-sandbox be specified?